test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -938,22 +938,6 @@ class TestDapp {
     );
   }
 
-  /**
-   * Sign a message with the signTypedDataV3 method.
-   */
-  async signTypedDataV3() {
```

Comment:
remove this deprecated method as we have `signTypedDataV3Redesign` in this class